src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/FilterFunctions.txt

```diff
@@ -33,7 +33,7 @@
 4
 
 >> Filter([1,2,0,3,4], 1/ThisRecord.Value >= 0)
-[1,2,3,4]
+Table({Value:1},{Value:2},Microsoft.PowerFx.Core.Public.Values.ErrorValue,{Value:3},{Value:4})
```

Comment:
>Microsoft.PowerFx.Core.Public.Values.ErrorValue [](http://example.com/codeflow?start=26&length=47)

this is the change

Reply:
Yes - all the other changes in the tests are related to the change in representation of tables that don't have a single column called 'Value'